pyvista/plotting/render_window_interactor.py

```diff
@@ -233,18 +233,64 @@ def enable_zoom_style(self):
         self._style_class = None
         return self.update_style()
 
-    def enable_terrain_style(self):
+    def enable_terrain_style(self, mouse_wheel_zooms=False, shift_pans=False):
         """Set the interactive style to terrain.
 
         Used to manipulate a camera which is viewing a scene with a natural
         view up, e.g., terrain. The camera in such a scene is manipulated by
         specifying azimuth (angle around the view up vector) and elevation
         (the angle from the horizon).
 
+        Parameters
+        ----------
+        mouse_wheel_zooms : bool, optional
+            Whether to use the mouse wheel for zooming. By default zooming
+            can be performed with right click and drag.
+
+        shift_pans : bool, optional
+            Whether shift + left mouse button pans the scene. By default
+            shift + left mouse button rotates the view restricted to only
+            horizontal or vertical movements, and panning is done holding
+            down the middle mouse button.
+
```

Comment:
Single request is to add an Example section to this.  One of the best things about `numpy` documentation is its examples.  Half the time I don't even read the parameter descriptions and just look to see how its used.

Reply:
Good idea. Let me try to do that for every style.

Reply:
Added in https://github.com/pyvista/pyvista/pull/1376/commits/377c61fd2e2449fb153729c025a51684f33d5dc5